2023-11-24 17:18:07,488 INFO    MainThread:29599 [wandb_init.py:_log_setup():292] Logging user logs to /content/drive/MyDrive/on-policy/onpolicy/scripts/results/StarCraft2/2m_vs_1z/rmappo/check/wandb/run-20231124_171807-3kq9dhdx/logs/debug.log
2023-11-24 17:18:07,490 INFO    MainThread:29599 [wandb_init.py:_log_setup():293] Logging internal logs to /content/drive/MyDrive/on-policy/onpolicy/scripts/results/StarCraft2/2m_vs_1z/rmappo/check/wandb/run-20231124_171807-3kq9dhdx/logs/debug-internal.log
2023-11-24 17:18:07,490 INFO    MainThread:29599 [wandb_setup.py:_flush():68] setting env: {}
2023-11-24 17:18:07,490 INFO    MainThread:29599 [wandb_setup.py:_flush():68] setting user settings: {}
2023-11-24 17:18:07,490 INFO    MainThread:29599 [wandb_setup.py:_flush():68] multiprocessing start_methods=fork,spawn,forkserver
2023-11-24 17:18:09,304 INFO    MainThread:29599 [wandb_run.py:_console_start():1265] atexit reg
2023-11-24 17:18:09,307 INFO    MainThread:29599 [wandb_run.py:_redirect():1135] redirect: SettingsConsole.REDIRECT
2023-11-24 17:18:09,309 INFO    MainThread:29599 [wandb_run.py:_redirect():1138] Redirecting console.
2023-11-24 17:18:09,309 INFO    MainThread:29599 [redirect.py:install():196] install start
2023-11-24 17:18:09,310 INFO    MainThread:29599 [redirect.py:install():211] install stop
2023-11-24 17:18:09,310 INFO    MainThread:29599 [redirect.py:install():196] install start
2023-11-24 17:18:09,313 INFO    MainThread:29599 [redirect.py:install():211] install stop
2023-11-24 17:18:09,313 INFO    MainThread:29599 [wandb_run.py:_redirect():1182] Redirects installed.
2023-11-24 17:18:11,191 INFO    Dummy-4   :29599 [wandb_run.py:_atexit_cleanup():1238] got exitcode: 1
2023-11-24 17:18:11,193 INFO    Dummy-4   :29599 [wandb_run.py:_restore():1210] restore
2023-11-24 17:18:11,193 INFO    Dummy-4   :29599 [redirect.py:uninstall():215] uninstall start
2023-11-24 17:18:11,194 INFO    Dummy-4   :29599 [redirect.py:_stop():264] _stop: stdout
2023-11-24 17:18:11,194 INFO    stdout    :29599 [redirect.py:_pipe_relay():119] relay done saw last write: stdout
2023-11-24 17:18:11,195 INFO    stdout    :29599 [redirect.py:_pipe_relay():135] relay done done: stdout
2023-11-24 17:18:11,195 INFO    Dummy-4   :29599 [redirect.py:_stop():270] _stop closed: stdout
2023-11-24 17:18:11,195 INFO    Dummy-4   :29599 [redirect.py:_stop():276] _stop joined: stdout
2023-11-24 17:18:11,195 INFO    Dummy-4   :29599 [redirect.py:_stop():278] _stop rd closed: stdout
2023-11-24 17:18:11,195 INFO    Dummy-4   :29599 [redirect.py:uninstall():219] uninstall done
2023-11-24 17:18:11,196 INFO    Dummy-4   :29599 [redirect.py:uninstall():215] uninstall start
2023-11-24 17:18:11,196 INFO    Dummy-4   :29599 [redirect.py:_stop():264] _stop: stderr
2023-11-24 17:18:11,197 INFO    Dummy-4   :29599 [redirect.py:_stop():270] _stop closed: stderr
2023-11-24 17:18:11,197 INFO    stderr    :29599 [redirect.py:_pipe_relay():119] relay done saw last write: stderr
2023-11-24 17:18:11,197 INFO    stderr    :29599 [redirect.py:_pipe_relay():135] relay done done: stderr
2023-11-24 17:18:11,197 INFO    Dummy-4   :29599 [redirect.py:_stop():276] _stop joined: stderr
2023-11-24 17:18:11,198 INFO    Dummy-4   :29599 [redirect.py:_stop():278] _stop rd closed: stderr
2023-11-24 17:18:11,198 INFO    Dummy-4   :29599 [redirect.py:uninstall():219] uninstall done
2023-11-24 17:18:11,210 INFO    Dummy-4   :29599 [wandb_run.py:_wait_for_finish():1349] got exit ret: uuid: "1f6e3c0fadb54edfa80b18865d6177fd"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 1311
      total_bytes: 1311
    }
  }
}

2023-11-24 17:18:13,217 INFO    Dummy-4   :29599 [wandb_run.py:_wait_for_finish():1349] got exit ret: uuid: "2c5a43f6508940f98553b9bc6c1c6525"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 10312
      total_bytes: 10312
    }
  }
}

2023-11-24 17:18:14,346 INFO    Dummy-4   :29599 [wandb_run.py:_show_files():1543] logging synced files
2023-11-24 17:18:14,351 INFO    Dummy-4   :29599 [internal.py:handle_exit():137] Internal process exited
